component/src/main/java/com/synopsys/integration/alert/component/audit/web/AuditEntryActions.java

```diff
@@ -91,7 +92,9 @@ public AuditEntryActions(
         if (!authorizationManager.hasReadPermission(ConfigContextEnum.GLOBAL, descriptorKey)) {
             return new ActionResponse<>(HttpStatus.FORBIDDEN, ActionResponse.FORBIDDEN_MESSAGE);
         }
-        AuditEntryPageModel pagedRestModel = auditAccessor.getPageOfAuditEntries(pageNumber, pageSize, searchTerm, sortField, sortOrder, onlyShowSentNotifications, auditAccessor::convertToAuditEntryModelFromNotification);
+        Integer page = ObjectUtils.defaultIfNull(pageNumber, 0);
+        Integer size = ObjectUtils.defaultIfNull(pageSize, 100);
```

Comment:
I think the class that `AuditEntryPageModel` extends might have constants that could be used for the default values.

Reply:
If those constants don't exist we should at least make some in this class

Reply:
I am now using the constants in AlertPagedModel. But that introduces a new issue https://github.com/blackducksoftware/blackduck-alert/pull/1249#discussion_r520829142